### PR TITLE
Adding empty state to execute modal without systems

### PR DESCRIPTION
**Pull request by arivepr**
This PR aims to close [this card](https://issues.redhat.com/browse/RHCLOUD-14910)

Screen Shot 2021-06-30 at 11 03 08 AM

_**Green square around x button is from trying to take screenshot**_

Also, any feedback to make the conditional rendering of the table portion is welcomed with open arms. Couldn't figure out a way to make it simpler/cleaner, and did not want to spin wheels on it :D

### CODE REVIEW

File: src/components/Modals/ExecuteModal.js [hunk 1]
```diff
@@ -149,42 +150,57 @@ export const ExecuteModal = ({
...
+              >
+                Download playbook
+              </Button>,
+              isDebug() ? (
+                <Button
+                  key="reset-etag"
+                  onClick={() => setEtag('test')}
+                  ouiaId="reset-etag"
+                >
+                  Reset etag
```
Comment (Hyperkid123) [inline src/components/Modals/ExecuteModal.js:190]: typo in `etag`?

**Review by Hyperkid123 — COMMENTED** [review]
Just found a little typo? Or is it supposed to be like that?

File: src/components/Modals/ExecuteModal.js (hunk 1 above)
Comment (arivepr) [reply src/components/Modals/ExecuteModal.js:190]: not sure what this is, I don't think I wrote that directly

Comment (ryelo) [reply src/components/Modals/ExecuteModal.js:190]: etag is correct

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**